.test-infra/jenkins/PythonTestProperties.groovy

```diff
@@ -17,6 +17,7 @@
  */
 
 class PythonTestProperties {
+  final static List<String> SUPPORTED_CONTAINER_TASKS = ['py36', 'py37', 'py38']
```

Comment:
@lazylynx FYI we could make use of the ALL_SUPPORTED_VERSIONS instead of this constant after your upcoming change (https://github.com/apache/beam/pull/13382).